admin-frontend/src/components/checkIn.js

```diff
@@ -88,8 +104,14 @@ const CheckInView = (props) => {
     showNotification('Jokin meni pieleen! Kokeile uudestaan.', 'warning')
   };
 
+  const notRedirectingBack = (location) => !isSubstring(location.pathname, "checkIn");
```

Comment:
Maybe a bit misleading name as it's checking against a constant path. So, "navigatingToCheckIn" or something?